packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -162,39 +162,48 @@ class MultipartUploader {
       return
     }
 
-    // For a 100MB file, with the default min chunk size of 5MB and a limit of 10:
-    //
-    // Total 20 parts
-    // ---------
-    // Need 1 is 10
-    // Need 2 is 5
-    // Need 3 is 5
-    const need = this.options.limit - this.partsInProgress
-    const completeChunks = this.chunkState.filter((state) => state.done).length
-    const remainingChunks = this.chunks.length - completeChunks
-    let minNeeded = Math.ceil(this.options.limit / 2)
-    if (minNeeded > remainingChunks) {
-      minNeeded = remainingChunks
-    }
-    if (need < minNeeded) return
+    const getChunkIndexes = () => {
```

Comment:
```suggestion
    function getChunkIndexes() {
```

functions give better stack traces, I think?

Reply:
Can't do that because we use `this` inside :)

Reply:
FWIW both syntaxes produces the same stack trace (and it would be the same with `const getChunkIndexes = function(){`).

Reply:
ahh the dreaded `this` keyword...